apps/Gruntfile.js

```diff
@@ -354,6 +354,7 @@ module.exports = function (grunt) {
     'levelbuilder_studio': './src/code-studio/levelbuilder_studio.js',
     'levelbuilder_gamelab': './src/code-studio/levelbuilder_gamelab.js',
     'levelbuilder_applab': './src/code-studio/levelbuilder_applab.js',
+    'levelbuilder_edit_script': './src/code-studio/levelbuilder_edit_script.js',
```

Comment:
@pcardune (or maybe @Bjvanminnen?) can probably shed some light on this.  I'm surprised React is ending up in this built file though.  Isn't it supposed to be extracted into a minimal.js file at some point?